libs/docs/platform/form-generator/examples/platform-form-generator-inline-help-example.component.html

```diff
@@ -13,10 +13,16 @@
     <fdp-button type="submit" label="Submit form"></fdp-button>
 </fdp-form-generator>
 
+<fdp-form-generator [formItems]="formItemsWithTemplateHint" mainTitle="Templates can be passed as hints">
+    <fdp-button type="submit" label="Submit form"></fdp-button>
+</fdp-form-generator>
+
 <fdp-form-generator
     [formItems]="formItemsHintOnGroupHeader"
     [hint]="formGeneratorMainHint"
     mainTitle="Form generator with hint"
 >
     <fdp-button type="submit" label="Submit form"></fdp-button>
 </fdp-form-generator>
+
+<ng-template #hintTemplate> Hint template! <fd-icon glyph="activate"></fd-icon> </ng-template>
```

Comment:
Would it be difficult to also pass configuration of the current form item? It might be useful to render text from label or value

Reply:
It is problematic because the template is for inline help and inline help does nothing to do with control in general. If someone requests it, we can do that as a separate feature

Reply:
Although, idea is cool